tests/wicked/before_test.pm

```diff
@@ -53,14 +53,19 @@ sub run {
     my $package_list = 'openvpn';
     $package_list .= ' tcpdump' if get_var('WICKED_TCPDUMP');
     if (check_var('IS_WICKED_REF', '1')) {
-        # Common REF Configuration
```

Comment:
why do you delete this info line?

Reply:
because I introduced if/else here so there is no more "common configuration of REF" :) so I thought it is irrelevant now . You disagree ?

Reply:
... but I can return back because looks like we need DHCPv4 for IPv6 testing too . so we **do** have common section :)